### DISCUSSION TITLE
when I use "collection.num_entities", but nothing return

### DISCUSSION THREAD

**Discussion by shantzhou** [milvus-io/milvus#7051] [Q&A and General discussion]
when I use "collection.num_entities", but nothing return

**Comment by yanliang567** [comment]
how many entities did you insert at a time?

**Comment by shantzhou** [comment]
ninety-eight
by the way:
&nbsp; &nbsp; in&nbsp; "https://github.com/milvus-io/bootcamp/blob/master/etc/onnx_and_milvus/milvus_with_different_models.ipynb"
When inserting the features of all pictures, the model should not be loaded every time
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "milvus-io/milvus"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2021年8月12日(星期四) 上午9:37
***@***.***&gt;;
抄送:&nbsp;"Shente ***@***.******@***.***&gt;;
主题:&nbsp;Re: [milvus-io/milvus] when I use "collection.num_entities", but nothing return (#7051)





 
how many entities did you insert at a time?
 
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Reply by yanliang567** [reply]
could you please specify which version you are using? cluster mode or standalone? sdk version? 
also please upload the logs if possible.
